print() is a function in Python 3

flake8 testing of https://github.com/lzfelix/pyStrings on Python 3.6.3

$ __flake8 . --count --select=E901,E999,F821,F822,F823 --show-source --statistics__
```
./levenshtein.py:302:64: E999 SyntaxError: invalid syntax
    print "Aligning %s and %s using Needleman-Wunsch algorithm:" % (s1, s2)
                                                               ^
1     E999 SyntaxError: invalid syntax
1
```

--- a/levenshtein.py
+++ b/levenshtein.py
@@ -299,19 +299,19 @@ if __name__ == "__main__":
     s1 = 'GATTACA'
     s2 = 'GCATGCU'
 
-    print "Aligning %s and %s using Needleman-Wunsch algorithm:" % (s1, s2)
-    print needleman_wunsch(s1, s2)
+    print("Aligning %s and %s using Needleman-Wunsch algorithm:" % (s1, s2))
+    print(needleman_wunsch(s1, s2))
 
     s1 = "evening"
     s2 = "seven"
 
-    print "Calculating Levenshtein (or edit) Distance between %s and %s" % (s1, s2)
+    print("Calculating Levenshtein (or edit) Distance between %s and %s" % (s1, s2))
 
     cost, edt1, edt2 = levenshtein_dp(s1, s2)
-    print edt1
-    print edt2
-    print 'Total cost (using DP): %d' % cost
+    print(edt1)
+    print(edt2)
+    print('Total cost (using DP): %d' % cost)
 
-    print 'Total cost (using abbreviated DP): %d' % levenshtein_short(s1, s2, False)
+    print('Total cost (using abbreviated DP): %d' % levenshtein_short(s1, s2, False))
 
-    print 'Total cost (using the recursive method): %d' % levenshtein_recursive(s1, s2)
+    print('Total cost (using the recursive method): %d' % levenshtein_recursive(s1, s2))

